sechub-server/src/main/resources/application.yml

```diff
@@ -44,6 +44,8 @@ spring:
   main:
     # see https://github.com/spring-projects/spring-boot/wiki/Spring-Boot-2.1-Release-Notes#bean-overriding
     allow-bean-definition-overriding: true
+    # see https://stackoverflow.com/questions/70036903/spring-boot-application-fails-to-start-after-upgrading-to-2-6-0-due-to-circular
+    allow-circular-references: true
```

Comment:
Ahem... 
The referenced stackoverflow entry is closed now because of missing debug information
https://stackoverflow.com/questions/70073748/i-cant-update-my-webapp-to-spring-boot-2-6-0-2-5-7-works-but-2-6-0-doesnt
is still open and has a same "solution".

But this can only be a temporary "solution"! We need to solve the problem. Having a circular reference can normally be fixed . So do we have already an github issue for this? Have you already tried to fix the problem with out the "solution" - workaround?

Reply:
No, I have not tried to fix the problem without the `allow-circular-references` set to `true` as I do not understand the issue well enough. I wanted to have your feedback on this. I opened a issue for this: #944.